### PR TITLE
feat: deploy showcase in github pages

### PR DESCRIPTION
**Pull request by fpaul-1A**
## Proposed change

https://fpaul-1a.github.io/otter

## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: .github/workflows/deploy-showcase.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    env:
+      NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
+    steps:
```
Comment (kpanot) [inline .github/workflows/deploy-showcase.yml:32]: Do you think it worth to deploy only if the app is touched?

**Comment by kpanot** [comment]
Let's merge it after redesign:
- [ ] we are happy with the look and field

File: .github/workflows/deploy-showcase.yml (hunk 1 above)
Comment (fpaul-1A) [reply .github/workflows/deploy-showcase.yml:32]: I would say yes, the app and its dependencies

Comment (fpaul-1A) [reply .github/workflows/deploy-showcase.yml:32]: this might not be as easy for a branch build

**Review by cpaulve-1A — APPROVED** [review]

**Review by vscaiceanu-1a — APPROVED** [review]

**Review by cpaulve-1A — APPROVED** [review]

**Pull request merged**